components/Contact/ContactFormStyleTwo.js

```diff
@@ -1,85 +1,79 @@
 import React, { useState } from 'react'
+import MailchimpSubscribe from "react-mailchimp-subscribe"
 
-const ContactFormStyleTwo = () => {
-    return (
-        <section className="main-contact-area pb-100">
-            <div className="container">
-                <div className="contact-wrap contact-pages mb-0">
-                    <div className="contact-form">
-                        <div className="section-title">
-                            <h2>Drop us a message for any query</h2>
-                        </div>
+const url = "https://us12.list-manage.com/contact-form?u=fdd40ebf441b888a2d75c41fd&form_id=eee2e8567ab48c31cc20a0349787ec09";
```

Comment:
looks like you have this in two places. Why are there Two ContactForm components?

Reply:
I'm unsure. I think this was created during the time when we worked with that temporary contributor on the contact page forms, can't get on the name now.